lib/active_model_serializers/rspec/matchers/schema.rb

```diff
@@ -0,0 +1,31 @@
+require 'active_model_serializers/test/schema'
+
+module ActiveModelSerializers
+  module RSpecMatchers
+    module Schema
+      extend ActiveSupport::Concern
+
+      included do
+        RSpec::Matchers.define :have_valid_schema do |schema_path|
```

Comment:
No need for a concern here.  Just define the matcher.  what's `Base`?  should use a fully qualified path.